homeassistant/components/hassio/coordinator.py

```diff
@@ -170,10 +169,10 @@ def get_core_info(hass: HomeAssistant) -> dict[str, Any] | None:
 
 @callback
 @bind_hass
-def get_issues_info(hass: HomeAssistant) -> SupervisorIssues | None:
+def get_issues_info(hass: HomeAssistant):
```

Comment:
Why was typing removed here?

Reply:
It created a circular import between the two. This seemed like the lesser of two evils. Any alternative suggestions? Ideally without a significant refactor.

Reply:
Wrap the import in a TYPE_CHECKING condition?

Reply:
Right, that would be easier haha, I always forget that option. Ok will do.